### PR TITLE
Update troubleshoot doc

### PR DESCRIPTION
**Pull request by boojamya**
This cleans up and reformats the doc.

It also adds a description around the `invalid header: new header has a time from the future` error.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
+- [Specific Errors](#specific-errors)
 
-**Ensure `rly` package is properly installed**
+<br>
+
+---
+
+<br>
+
+## Generic Troubleshoot
```
Comment (jtieri) [inline docs/troubleshooting.md]: Should this read, `Generic Troubleshooting`?

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -10,9 +21,8 @@
 
    If this returns an error, make sure you have Go installed and your Go environment is setup. Then redo [Step 1](#basic-usage---relaying-packets-across-chains).
 
----
 
-## **Verify valid `chain`, `client`, and `connection`**
+### **Verify valid `chain`, `client`, and `connection`**
```
Comment (jtieri) [inline docs/troubleshooting.md]: L25 and L36 are identical. It seems like this line may be supposed to read something along the lines of, "Verify valid `Chain`, `Keys`, `Balance` and `Path`

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -23,7 +33,7 @@ Output should show all checkboxes:
 -> type(cosmos) key(✔) bal(✔) path(✔)
 ```
 
-## **Verify valid `chain`, `client`, and `connection`**
+### **Verify valid `chain`, `client`, and `connection`**
```
Comment (jtieri) [inline docs/troubleshooting.md:36]: This line seems like it should read something along the lines of, "Verify valid `Clients`, `Connections`, and `Channels`"

File: docs/troubleshooting.md [hunk 4]
```diff
@@ -41,31 +51,53 @@ If:
...
+
 ---
 
-**Error querying blockdata**
+<br>
+
+## Specific Errors
+
+<details>
+<summary>error querying blockdata</summary>
```
Comment (jtieri) [inline docs/troubleshooting.md]: Should `blockdata` be two words?

**Review by jtieri — COMMENTED** [review]
good updates overall, just had a few comments

File: docs/troubleshooting.md (hunk 3 above)
Comment (boojamya) [reply docs/troubleshooting.md:36]: I had to double check this too, but this should indeed be `chain`.
It queries the latest height. If it returns an error, then chns(✘).

Added an extra note about this.

File: docs/troubleshooting.md (hunk 4 above)
Comment (boojamya) [reply docs/troubleshooting.md]: Yes!

**Review by jtieri — APPROVED** [review]

**Pull request merged**